package.json

```diff
@@ -141,16 +215,15 @@
     "web:build-examples": "cd website && node build-examples.js",
     "web:build": "npm-run-all web:inject-bundles-misc web:generate web:build-examples web:inject-frontpagecodesample",
     "web:bundle-watch-inject": "onchange 'packages/uppy/dist/**/*.css' 'packages/uppy/dist/**/*.js' --initial --verbose -- npm run web:inject-bundles-misc",
-    "web:clean": "cd website && touch db.json && ./node_modules/.bin/hexo clean",
+    "web:clean": "cd website && touch db.json && ../node_modules/.bin/hexo clean",
```

Comment:
Don’t we run these `web:` scripts from root? Why `..`?

Reply:
Because of the `cd` at the start of the line, so ./ is in the website/ folder, but I'll remove the `cd` so it'll be less confusing

Reply:
Oh, right, sorry! Forgot about it :)

Reply:
Nevermind, we probably need the `cd` so that the working directory is correctly set to `./website` when Hexo runs :D